### PR TITLE
Mwpw 134073 table alignment fixed in mobile windows.

### PR DESCRIPTION
**Pull request by sharath-kannan**
*Min-width for mobile window has been added. 

Resolves: [MWPW-134073](https://jira.corp.adobe.com/browse/MWPW-134073)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/drafts/shrinidhivp/tables-test
- After: https://mwpw-134073--milo--sharath-kannan.hlx.page/drafts/shrinidhivp/tables-test?martech=off

### CODE REVIEW

File: libs/blocks/table/table.css [hunk 1]
```diff
@@ -443,3 +443,19 @@ header.global-navigation {
   width: 30px;
   margin-right: 16px;
 }
+
+/* Start mobile styles */
+@media (max-width: 400px) {
+  .section{
```
Comment (mokimo) [inline libs/blocks/table/table.css:449]: .section sounds really generic, I'm not sure the table should set something like that

Comment (sharath-kannan) [reply libs/blocks/table/table.css]: Have changed it to a specific selector now.

File: libs/blocks/table/table.css [hunk 2]
```diff
@@ -443,3 +443,16 @@ header.global-navigation {
   width: 30px;
   margin-right: 16px;
 }
+
+/* Start mobile styles */
```
Comment (chrischrischris) [inline libs/blocks/table/table.css:447]: No need for this comment

**Review by chrischrischris — APPROVED** [review]

File: libs/blocks/table/table.css (hunk 2 above)
Comment (sharath-kannan) [reply libs/blocks/table/table.css]: done.

**Review by mokimo — APPROVED** [review]

**Pull request merged**